packaging/hipify-clang.rpm_postun

```diff
@@ -0,0 +1,8 @@
+ROCMDIR="/opt/rocm"
+HIPDIR=$ROCMDIR/hip
+HIPBINDIR=$HIPDIR/bin
```

Comment:
HIPBINDIR=/opt/rocm/bin

Reply:
Sure, can do that. Was just keeping up with convention in https://github.com/ROCm-Developer-Tools/hipamd/blob/develop/packaging/hip-devel.postinst